src/common/components/Navbar/NavbarLogo.tsx

```diff
@@ -6,20 +6,20 @@ type navbarLogoProps = {
 
 const NavbarLogo: FC<navbarLogoProps> = ({ navbarLogo }) => {
   return (
-    <div className="flex mt-5 ">
-      <div className="w-[15px] h-auto bg-logoColor"></div>
-      <div className="bg-logoColor w-[180px] max-h-[100px] relative">
+    <div className="xs:w-full xs:mt-5 mt-0 flex w-2/3 sm:w-full">
```

Comment:
Isn't the "xs" useless in this case? It will apply the same if you only leave the "sm:w-full"

Reply:
Fixed, not needed was "sm"